src/components/Autosuggest/Autosuggest.scss

```diff
@@ -8,6 +8,10 @@
     &--focused {
         position: absolute;
         width: 100%;
+
+        .Autosuggest__field {
```

Comment:
Nesting is not necessary here, can be `.Autosuggest__field--focused`.

Reply:
changed it